src/player.cpp

```diff
@@ -7683,36 +7685,38 @@ ret_val<bool> player::can_wear( const item& it  ) const
         !it.has_flag( "OVERSIZE" ) &&
         is_wearing_helmet() ) {
         return ret_val<bool>::make_failure( wearing_something_on( bp_head ),
-                                            _( "You can't wear that with other headgear!" ) );
+                                            ( is_player() ? _( "You can't wear that with other headgear!" )
+                                             : string_format( _( "%s can't wear that with other headgear!" ), name.c_str() ) ) );
     }
 
     if( it.covers( bp_head ) &&
         ( it.has_flag( "SKINTIGHT" ) || it.has_flag( "HELMET_COMPAT" ) ) &&
         ( head_cloth_encumbrance() + it.get_encumber() >= 20 ) ) {
-        return ret_val<bool>::make_failure( _( "You can't wear that much on your head!" ) );
+        return ret_val<bool>::make_failure( ( is_player() ? _( "You can't wear that much on your head!" )
+                                              : string_format( _( "%s can't wear that much on their head!" ), name.c_str() ) ) );
     }
 
     if( has_trait( trait_WOOLALLERGY ) && ( it.made_of( material_id( "wool" ) ) || it.item_tags.count( "wooled" ) ) ) {
-        return ret_val<bool>::make_failure( _( "You can't wear that, it's made of wool!" ) );
+        return ret_val<bool>::make_failure( _( "Can't wear that, it's made of wool!" ) );
     }
 
     if( it.is_filthy() && has_trait( trait_SQUEAMISH ) ) {
-        return ret_val<bool>::make_failure( _( "You can't wear that, it's filthy!" ) );
+        return ret_val<bool>::make_failure( _( "Can't wear that, it's filthy!" ) );
     }
 
     if( !it.has_flag( "OVERSIZE" ) ) {
         for( const trait_id &mut : get_mutations() ) {
             const auto &branch = mut.obj();
             if( branch.conflicts_with_item( it ) ) {
-                return ret_val<bool>::make_failure( _( "Your mutation %s prevents you from wearing that %s." ),
+                return ret_val<bool>::make_failure( _( "Mutation %s prevents from wearing %s." ),
```

Comment:
`Mutation %s prevents from wearing %s` reads like this:
"Mutation Beak prevents from wearing chitin helmet".
Which is a bit stilted.

Suggest `The %s mutation prevents wearing [a/the] %s.` so it reads like:
"The Beak mutation prevents wearing a chitin helmet."
"The Beak mutation prevents wearing the chitin helmet."